tools/cldr-apps/js/src/views/DashboardWidget.vue

```diff
@@ -26,7 +26,7 @@
           title="Close"
           @click="closeDashboard"
         >
-          x
+          X
```

Comment:
better as `glyphicon-remove-sign` ?

Reply:
maybe -- let's discuss, as it relates to bootstrap, fonts...